### PR TITLE
[5.x] provide patch operations for json field

### PR DESCRIPTION
**Pull request by vangheem**

### CODE REVIEW

File: guillotina/db/interfaces.py [hunk 1]
```diff
@@ -328,6 +328,7 @@ async def terminate(conn):
 
 class IPostgresStorage(IStorage):
     objects_table_name = Attribute("")
+    blobs_table_name = Attribute("")
```
Comment (bloodbare) [inline guillotina/db/interfaces.py:331]: Is this related to this PR?

Comment (vangheem) [reply guillotina/db/interfaces.py:331]: ahh, nah, I was in the mode of adding some mypy for some code. I can remove it.
